services/messages-service.ts

```diff
@@ -0,0 +1,88 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import * as util from "util";
+import * as path from "path";
+
+export class MessagesService implements IMessagesService {
+	private _pathsToMessageJsonFiles: string[] = null;
+	private _messageJsonFilesContentsCache: any[] = null;
+
+	private get pathToDefaultMessageJson(): string {
+		return path.join(path.dirname(__dirname), "resources", "messages", "errorMessages.json");
```

Comment:
Sadly `$resources` is not registered anywhere in the boundaries of the common library so I can't `import` it